README.md

```diff
@@ -4,84 +4,89 @@ xhgui
 A graphical interface for XHProf data built on MongoDB.
 
 This tool requires that [XHProf](http://pecl.php.net/package/xhprof) or 
-its fork [Uprofiler](https://github.com/FriendsOfPHP/uprofiler) is installed, 
-which is a PHP Extension that records and provides profiling data.
+its fork [Uprofiler](https://github.com/FriendsOfPHP/uprofiler) is installed.
+XHProf is a PHP Extension that records and provides profiling data.
 XHGui (this tool) takes that information, saves it in MongoDB, and provides
-a convienent GUI for working with it.
-
+a convenient GUI for working with it.
 
 System Requirements
 ===================
 
+XHGui has the following requirements:
+
  * [XHProf](http://pecl.php.net/package/xhprof) or 
-   [Uprofiler](https://github.com/FriendsOfPHP/uprofiler) to actually profile the data
- * [MongoDB PHP](http://pecl.php.net/package/mongo) MongoDB PHP extension
- * [MongoDB](http://www.mongodb.org/) MongoDB Itself
- * [mcrypt] (http://php.net/manual/en/book.mcrypt.php) PHP must be configured
-   with mcrypt (which is a dependency of Slim)
- * [dom] (http://php.net/manual/en/book.dom.php) If you are running the tests
-   you'll need the DOM extension (which is a dependency of PHPUnit)
+   [Uprofiler](https://github.com/FriendsOfPHP/uprofiler) to actually profile the data.
+ * [MongoDB PHP](http://pecl.php.net/package/mongo) MongoDB PHP driver.
+   XHGui requires verison 1.3.0 or later.
+ * [MongoDB](http://www.mongodb.org/) MongoDB Itself.
+ * [mcrypt](http://php.net/manual/en/book.mcrypt.php) PHP must be configured
+   with mcrypt (which is a dependency of Slim).
+ * [dom](http://php.net/manual/en/book.dom.php) If you are running the tests
+   you'll need the DOM extension (which is a dependency of PHPUnit).
 
 
 Installation
 ============
 
-Installing Xhgui requires 2 main steps. First is installing the `xhgui`
-front-end, and the second is profiling a web application/site.
+1. Clone or download `xhgui` from Github.
 
+2. Point your webserver to the `webroot` directory.
 
-Installing Xhgui
-----------------
+3. Set the permissions on the `cache` directory to allow the
+   webserver to create files. If you're lazy, `0777` will work.
+   
+   The following command changes the permissions for the `cache` directory:
 
-* Clone or download `xhgui` from github.
-* You'll need to install `mongodb`, and `php-mongodb`, at least version 1.3.0
-  of the php extension is required.
-* Point your webserver to the `webroot` directory.
-* Set the permissions on the `cache` cache directory to allow the webserver to
-  create files.  If you're lazy `0777` will work. Run:
+   ```
+   chmod -R 0777 cache
+   ```
 
-  ```
-  chmod -R 0777 cache
-  ```
+4. Start a MongoDB instance. XHGui uses the MongoDB instance to store
```

Comment:
This wasn't really alluded to previously, so I added it explicitly before talking about configuring to connect to the `mongod` / setting up indexes.